app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -505,6 +510,32 @@ private void saveImagesInGrayscale() {
         }
     }
 
+    /**
+     * Returns the last file name obtained from filepath
+     */
+    private String getLastFileName(ArrayList<String> filesPath) {
+
+        if (filesPath.size() == 0)
+            return "";
+        String lastSelectedFilePath = filesPath.get(filesPath.size() - 1);
+        String nameWithoutExt = stripExtension(getFileNameWithoutExtension(lastSelectedFilePath));
+        return nameWithoutExt + getString(swati4star.createpdf.R.string.pdf_suffix);
+    }
+
+    private String stripExtension(String str) {
```

Comment:
please add javadoc style comments to newly added functions

Reply:
yup added javadoc.